kubernetes/policy-anyOf.rego

```diff
@@ -0,0 +1,8 @@
+package kubernetes.admission                                                
+
+default deny = true 
+deny {                                                                
```

Comment:
What do you think about adding METADATA to `policy-anyOf.rego`  both in the file and in the snippet you have in `schemas.md`  to be a little more explicit of schema usage for those uploading a directory of schemas?

I assume you ran `opa eval .... -s kubernetes/schemas/input-anyOf.json`  so it used the input schema file as default

Before line 4, you can maybe add:
```
# METADATA
# scope: rule
# schemas: 
#   - input: schema["input-anyOf"]
```
This should work with `opa eval .... -s kubernetes/schemas` - maybe you should try it to confirm?